### ISSUE TITLE
Some browser features cause JVM crash on Linux 64bit

### ISSUE THREAD

**Issue by msrocka** [GreenDelta/olca-app#4]
Some browser features (JavaScript calls) cause a JVM crash on Linux 64bit (Lubuntu). This can be reproduced by opening the KML map editor as an example. Pages like the start page work.

Error log: 

```
    # A fatal error has been detected by the Java Runtime Environment:
    #
    # SIGSEGV (0xb) at pc=0x00007f951596b4cf, pid=2958, tid=140278291138304
    #
    # JRE version: Java(TM) SE Runtime Environment (8.0_05-b13) (build 1.8.0_05-b13)
    # Java VM: Java HotSpot(TM) 64-Bit Server VM (25.5-b02 mixed mode linux-amd64
         compressed     oops)
    # Problematic frame:
    # C [ld-linux-x86-64.so.2+0xc4cf] _dl_rtld_di_serinfo+0x2dbf
    #
```

**Comment by msrocka** [comment]
We will switch to JavaFX for the webviews in openLCA 1.6 which should solve this issue.

**Comment by msrocka** [comment]
fixed it in openLCA 1.6.* with the JavaFX webviews
